packages/@romefrontend/compiler/lint/rules/js/noNestedTernary.ts

```diff
@@ -0,0 +1,23 @@
+import {Path, TransformExitResult} from "@romefrontend/compiler";
+import {descriptions} from "@romefrontend/diagnostics";
+
+export default {
+	name: "jsNoNestedTernary",
+	enter(path: Path): TransformExitResult {
+		const {node} = path;
+
+		if (node.type === "JSConditionalExpression") {
+			if (
+				node.alternate.type === "JSConditionalExpression" ||
+				node.consequent.type === "JSConditionalExpression"
+			) {
+				path.context.addNodeDiagnostic(
+					node,
+					descriptions.LINT.JS_NO_NESTED_TERNARY,
+				);
+			}
```

Comment:
I think it'd be clearer if the error was displayed on the problematic ternary rather than the "outer" one.

```suggestion
			if (node.alternate.type === "JSConditionalExpression") {
				path.context.addNodeDiagnostic(
					node: node.alternate,
					descriptions.LINT.JS_NO_NESTED_TERNARY,
				);
			}

			if (node.consequent.type === "JSConditionalExpression") {
				path.context.addNodeDiagnostic(
					node: node.consequent,
					descriptions.LINT.JS_NO_NESTED_TERNARY,
				);
			}
```